pages/gallery.html

```diff
@@ -2,11 +2,60 @@
 <html lang="en">
 <head>
   <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <title>Gallery | Pizza Fans United</title>
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  <link rel="stylesheet" href="../styles/gallery.css">
+  
 </head>
 <body>
-  
+  <header>
+    <h1>Pizza Gallery</h1>
+  </header>
+    <nav class="main-nav">
+      <ul>
+        <li><a href="../index.html">Home</a></li>
+        <li><a href="facts.html">Fun Facts</a></li>
+        <li><a href="gallery.html">Gallery</a></li>
+        <li><a href="faq.html">Frequently Asked Questions</a></li>
+      </ul>
+    </nav>
+
+  <main>
+    <section class="container">
+      <figure>
+        <img src="https://pagliacci.imgix.net/Mobile/Online/Pizza/Margherita_2018_H.jpg?w=720" alt="Margherita pizza with basil leaves">
+        <figcaption>
+          <strong>Classic Margherita</strong> – fresh basil, mozzarella & tomato<br>
+          <a href="https://www.pagliacci.com/order/products/6559" target="_blank">Image source</a>
+        </figcaption>
+      </figure>
+
+      <figure>
+        <img src="https://pagliacci.imgix.net/Mobile/Online/Pizza/Extra_Pepperoni_2018_H.jpg?w=720" alt="Pepperoni pizza slice">
+        <figcaption>
+          <strong>Pepperoni Perfection</strong> – loaded with pepperoni slices<br>
```

Comment:
We want to be pretty sparing with our use of the `br` tag and try to save it for places where a line break is really meaningful to the content (Examples from the MDN docs on `br` are poems and postal addresses). It's considered better form to use CSS to set margins or create space around an element.

A primary reason to avoid `br`s is that they can make navigation with screen readers more difficult. I recommend taking a look at the accessibility concerns section of the MDN's `br` reference. Their suggested replacement is to use individual `<p>` elements and use styling on properties like the margin to create space between them. 
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br#accessibility_concerns